js/background.js

```diff
@@ -1,9 +1,25 @@
-/*
- * vim: ts=4:sw=4:expandtab
- */
+/* eslint-disable */
+
+/* eslint-env browser */
```

Comment:
What if we put a `.eslintignore` in the `/js` directory that had the env and some of these globals set already? Then we won't need to put this in every file we process with eslint.

Also, let's think about the right way to prepare for a future where we use `require()` for all dependencies. What if we forced all global references to the top of the file?